manifests/permissive.pp

```diff
@@ -19,10 +20,7 @@
   Selinux::Permissive[$title] ->
```

Comment:
removing would need to happen before removing modules because one can not remove a module which has it set to permissive with semanage permissive.